generators/generator-defaults.js

```diff
@@ -96,6 +96,12 @@ const translationDefaultConfig = {
   },
 };
 
+const kubernetesDefaultConfig = {
+  kubernetesServiceType: 'LoadBalancer',
+  monitoring: 'no',
+  istio: false,
+};
+
```

Comment:
Thinking about modularization, I think we should move to the Kubernetes folder, like I did with init generator.
You can import here and reexport for a clean api, this is the new standard for esm/node.